addons/mail/static/src/core/web/message_patch.js

```diff
@@ -79,6 +79,7 @@ patch(Message.prototype, {
     },
 
     openRecord() {
+        this.message.setDone();
```

Comment:
I believe AJU wants ALL the `user_notification` messages currently visible in inbox that are related to `this.message.thread` to be marked as done..